src/Databases/MySQL/MaterializedMySQLSyncThread.cpp

```diff
@@ -817,6 +820,7 @@ void MaterializedMySQLSyncThread::executeDDLAtomic(const QueryEvent & query_even
 
         String query = query_event.query;
         tryQuoteUnrecognizedTokens(query, query);
+        tryConvertStringLiterals(query, query);
```

Comment:
as I see everywhere apart from gtest this function gets the same variable as first and second argument, what is the point then? May be just have one `String & query` argument instead of two? Because it looks a bit confusing now. Probably the same is with tryQuoteUnrecognizedTokens.